### PR TITLE
Leader Componente

### LINKED ISSUE #2: Leader Member Component
# Leader Member Component

- [X] Mockup
- [ ] Code
- [ ] Review
- [ ] Integration

### Mockup Img

Image

### PR DESCRIPTION
**Pull request by Felipe34515**
image
Closes #2

### CODE REVIEW

File: src/components/codeHome.astro
Comment (Crola1702) [inline src/components/codeHome.astro:1]: Why adding `codeHome` file for the `Leader` component?

File: src/pages/LiderDetail.astro [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
...529 6.75293 18.8379 8.05664 18.7109 8.37402C19.4922 9.23828 19.9707 9.91211 19.9707 11.25C19.9707 15.9619 17.0947 16.3379 14.3652 16.6455C14.8145 17.0312 15.1953 17.7637 15.1953 18.9111C15.1953 20.5566 15.1807 22.5928 15.1807 22.9932C15.1807 23.3105 15.4053 23.6963 16.0254 23.584C20.9082 21.9629 24.2188 17.3291 24.2188 11.9141C24.2188 5.1416 18.7256 0 11.9531 0ZM4.74609 16.8408C4.68262 16.8896 4.69727 17.002 4.78027 17.0947C4.8584 17.1729 4.9707 17.207 5.03418 17.1436C5.09766 17.0947 5.08301 16.9824 5 16.8896C4.92188 16.8115 4.80957 16.7773 4.74609 16.8408ZM4.21875 16.4453C4.18457 16.5088 4.2334 16.5869 4.33105 16.6357C4.40918 16.6846 4.50684 16.6699 4.54102 16.6016C4.5752 16.5381 4.52637 16.46 4.42871 16.4111C4.33105 16.3818 4.25293 16.3965 4.21875 16.4453ZM5.80078 18.1836C5.72266 18.2471 5.75195 18.3936 5.86426 18.4863C5.97656 18.5986 6.11816 18.6133 6.18164 18.5352C6.24512 18.4717 6.21582 18.3252 6.11816 18.2324C6.01074 18.1201 5.86426 18.1055 5.80078 18.1836ZM5.24414 17.4658C5.16602 17.5146 5.16602 17.6416 5.24414 17.7539C5.32227 17.8662 5.4541 17.915 5.51758 17.8662C5.5957 17.8027 5.5957 17.6758 5.51758 17.5635C5.44922 17.4512 5.32227 17.4023 5.24414 17.4658Z" fill="#FFD02A"/>
```
Comment (Crola1702) [inline src/pages/LiderDetail.astro:132]: What are these big numbers in the path?

File: src/pages/index.astro
Comment (Crola1702) [inline src/pages/index.astro:1]: Why changing the whole index page for the `Leader` component?

File: src/pages/ProyectosEnCurso.astro
Comment (Crola1702) [inline src/pages/ProyectosEnCurso.astro:1]: Why adding current project for in the `Leader` component?

**Review by Crola1702 — CHANGES_REQUESTED** [review]
Just some questions before approving

File: src/pages/index.astro
Comment (Felipe34515) [reply src/pages/index.astro:1]: There was a problem, apparently, when I made the animation of the Homepage and Project website, it was saved in the Pull request of leader component. I changed the index for the HomePage.

File: src/pages/ProyectosEnCurso.astro
Comment (Felipe34515) [reply src/pages/ProyectosEnCurso.astro:1]: There was a problem, apparently, when I made the animation of the Homepage and Project website, it was saved in the Pull request of leader component. I changed the index for the HomePage.

**Review by Crola1702 — APPROVED** [review]
LGTM

**Comment by Crola1702** [comment]
After conflicts resolved

**Comment by mr-sosa** [comment]
Finally, don't forget resolved the merge conflicts. You must leave the index.astro of your branch as it's in the branch develop

**Pull request merged**